addons/website/static/src/js/website.snippets.animation.js

```diff
@@ -328,7 +328,7 @@ animation.registry.parallax = animation.Class.extend({
         if (!this.$bg || !this.$bg.length) {
             this.$bg = this.$("> .s_parallax_bg");
             if (!this.$bg.length) {
-                this.$bg = $("<span/>", {"class": "s_parallax_bg"}).prependTo(this.$target);
+                this.$bg = $("<span/>", {"class": this.$target.hasClass('oe_custom_bg') ? "s_parallax_bg oe_custom_bg" : "s_parallax_bg"}).prependTo(this.$target);
```

Comment:
To prevent losing custom background on parallax slider ( issue : https://youtu.be/JNfBWiR6ffM?t=21s )